Week1/Section3/Exercise2/Source2.cpp

```diff
@@ -66,32 +66,32 @@ int main()
 		"Is a hot dog a taco though? "
 	};
 
-	bool Answers[3] = {
+	bool bAnswers[3] = {
```

Comment:
It may be confusing with the standards hehe, but since this is not a bool, but rather an array, it should not start with a b.

The `b` indicates "this variable is a bool" so you can use it as a bool. That is not the case with an array of bools.

Reply:
jajaj understandable 👌